extras/systemd/foreman.service

```diff
@@ -8,7 +8,7 @@ Type=simple
 User=foreman
 TimeoutSec=300
 WorkingDirectory=/usr/share/foreman
-ExecStart=/usr/bin/scl enable tfm "rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND"
+ExecStart=/usr/bin/foreman-ruby "rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND"
```

Comment:
This also breaks:
```
● foreman.service - Foreman
   Loaded: loaded (/usr/lib/systemd/system/foreman.service; disabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/foreman.service.d
           └─override.conf
   Active: failed (Result: exit-code) since Mon 2019-04-15 12:51:57 UTC; 34s ago
     Docs: https://theforeman.org
  Process: 1061 ExecStart=/usr/bin/foreman-ruby rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND (code=exited, status=1/FAILURE)
 Main PID: 1061 (code=exited, status=1/FAILURE)

Apr 15 12:51:57 debian9 systemd[1]: Started Foreman.
Apr 15 12:51:57 debian9 foreman-ruby[1061]: /usr/bin/foreman-ruby: No such file or directory -- rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND (LoadError)
Apr 15 12:51:57 debian9 systemd[1]: foreman.service: Main process exited, code=exited, status=1/FAILURE
Apr 15 12:51:57 debian9 systemd[1]: foreman.service: Unit entered failed state.
Apr 15 12:51:57 debian9 systemd[1]: foreman.service: Failed with result 'exit-code'.
```

For Debian it's bundled so `/usr/bin/bundle exec "..."` is needed. Not sure what a sane default would look like.

Reply:
We could default to SCL then and replace in debian? `/bin/tfm-ruby` or we can just drop this from the codebase and let packaging deploy the correct service file.

Reply:
I'm leaning to `/usr/bin/rails` and replacing it in packaging for EL7 and Debian. EL8 will probably package it so it looks native.

Reply:
Updated to this pathing